security/acme-client/src/opnsense/mvc/app/views/OPNsense/AcmeClient/certificates.volt

```diff
@@ -300,7 +283,9 @@ POSSIBILITY OF SUCH DAMAGE.
             {
                 if (gridParams['del'] != undefined) {
                     var uuid=$(this).data("row-id");
-                    stdDialogRemoveItem('Remove selected item?',function() {
+                    stdDialogConfirm('{{ lang._('Confirm removal') }}',
+                        '{{ lang._('Do you want to remove the selected item?') }}',
```

Comment:
I was forced to copy this code from `opnsense_bootgrid_plugin.js` and I don't want it to diverge from the original. Maybe at some point in the future this duplicate code can be removed.